spec/requests/content_pages_spec.rb

```diff
@@ -46,35 +48,32 @@
   end
 
   describe 'GET /modules/:training_module_id/content-pages/1-3-4' do
-    before do
-      get training_module_content_page_path(:alpha, '1-3-4')
-    end
-
     context 'when module is not completed' do
       it 'shows not completed' do
+        get training_module_content_page_path(:alpha, '1-3-4')
         expect(response.body).to include('You have not yet completed the module.')
       end
 
       it 'does not have the users name' do
+        get training_module_content_page_path(:alpha, '1-3-4')
         expect(response.body).not_to include('John Doe')
       end
     end
 
     context 'when module is completed' do
-      include_context 'with progress'
-
       before do
         view_whole_module(alpha)
```

Comment:
Can we move the get to the end of the `before` block

Reply:
get?